### PR TITLE
[SDKS-6093] Add e2e tests for redis optimized and none strategy

### PR DESCRIPTION
**Pull request by emmaz90**
# JS SDK

## What did you accomplish?
Added E2E tests for optimized and none strategy in redis
Fix tests for not counting first impression

## Extra Notes

### CODE REVIEW

File: src/__tests__/consumer/node_redis.spec.js [hunk 1]
```diff
@@ -146,6 +148,242 @@ tape('NodeJS Redis', function (t) {
...
+      });
+  });
+  
+  t.test('Regular usage - NONE strategy', assert => {
+    config.sync.impressionsMode = NONE;
+    initializeRedisServer()
+      .then(async (server) => {
+        const expectedConfig = '{"color":"brown"}';
+        const timeFrame = Date.now();
+        const expectedImpressionCount = [
```
Comment (EmilianoSanchez) [inline src/__tests__/consumer/node_redis.spec.js:277]: Nice work Emma!
Just a single comment.

If `expectedImpressionCount` is the same for the NONE and the OPTIMIZED tests, we can move it to line 71 and reuse in both tests. Same with `expectedUniqueKeys` and `expectedConfig` (this one is used 3 times)

**Review by EmilianoSanchez — APPROVED** [review]
